### PR TITLE
[TestTool] Provide performance tests for all performance envelopes

### PR DESCRIPTION
**Pull request by fabiotudone-da**
Performance tests are provided for all envelopes (but CI still runs only Beta)

CHANGELOG_BEGIN
[TestTool] Provide performance tests for all performance envelopes
CHANGELOG_END

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PerformanceEnvelope.scala [hunk 1]
```diff
@@ -37,7 +37,7 @@ import scala.util.{Failure, Success, Try}
 import com.daml.ledger.test.performance.{PingPong => PingPongModule}
 import org.slf4j.Logger
 
-case class Envelope(name: String, transactionSizeKb: Int, throughput: Int, latencyMs: Int)
+sealed case class Envelope(name: String, transactionSizeKb: Int, throughput: Int, latencyMs: Int)
```
Comment (stefanobaghino-da) [inline ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PerformanceEnvelope.scala:40]: `case class`es should never be inherited. If anything, I'd recommend making this `final` rather. `sealed` makes little sense.

Comment (fabiotudone-da) [reply ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PerformanceEnvelope.scala:40]: Right, the enum-avoiding (sigh) "recommended path" is now `sealed trait` I believe.

Comment (stefanobaghino-da) [reply ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PerformanceEnvelope.scala:40]: Either that, or simply make `Envelope` a `sealed abstract class`. Do you really need it to be a `case class`? The reason why `case class`es should be `final` in vast majority of cases (no pun intended) is to [make sure equality makes sense](https://stackoverflow.com/questions/11158929/what-is-so-wrong-with-case-class-inheritance/11159036#11159036). With [the notable exception of `sealed abstract case class`es](https://gist.github.com/tpolecat/a5cb0dc9adeacc93f846835ed21c92d2).

Comment (fabiotudone-da) [reply ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PerformanceEnvelope.scala]: It was a leftover when the code was ported. Aligned with the recommended pattern.

**Pull request merged**